src/sentry/utils/email.py

```diff
@@ -35,7 +35,7 @@
 from sentry.utils.strings import is_valid_dot_atom
 from sentry.web.helpers import render_to_string
 
-logger = logging.getLogger(__name__)
+logger = get_logger()
```

Comment:
If we didn't specify the name later, what ends up getting logged? I think it's worth having a sane fallback default name.